components/ChallengeQuestionCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import '../scss/challengeQuestionCard.scss'
+
+type Props = {
+  title: string
+  question: string
+}
+
+const ChallengeQuestionCard: React.FC<Props> = props => {
+  return (
+    <div className="card shadow-sm border-0">
+      <div className="card-body">
+        <h1 className="challenge-question-card__title">{props.title}</h1>
```

Comment:
could this just all be `{content}` we will use markdown to generate the HTML content.